hedera-node/src/main/java/com/hedera/services/store/models/Token.java

```diff
@@ -81,20 +105,236 @@
 	private JKey freezeKey;
 	private JKey supplyKey;
 	private JKey wipeKey;
-    private JKey adminKey;
-    private boolean frozenByDefault;
+	private JKey adminKey;
+	private boolean frozenByDefault;
 	private Account treasury;
 	private Account autoRenewAccount;
 	private boolean deleted;
 	private boolean autoRemoved = false;
 	private long expiry;
-
 	private long lastUsedSerialNumber;
+	static Predicate<Key> REMOVES_ADMIN_KEY = ImmutableKeyUtils::signalsKeyRemoval;
+
+	/* New properties. The comment is related to Yoan's expectations for merge conflicts */
+	private JKey feeScheduleKey;
+	private long autoRenewPeriod;
+	private String symbol;
+	private String name;
+	private String memo;
+
 
 	public Token(Id id) {
 		this.id = id;
 	}
 
+	//TODO: Refactor
+	//TODO: Consult with Yoan does the order of assertions is part of the business logic (in the previous implementation the expiry validation
```

Comment:
Yes, it is. The order of asserrtions is important, as we must **not** change the API